.github/workflows/ci-testing-deploy.yml

```diff
@@ -165,8 +165,8 @@ jobs:
           flags: unittests #optional
       - name: prepare codeclimate coverage file
         run: |
-          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-          chmod +x ./cc-test-reporter
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-0.7.0-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter && ./cc-test-reporter --version
```

Comment:
minor: not sure if it is possible to define a variable in the workflow file. it would prevent all this copy-pasting